### ISSUE TITLE
mount.nfs: requested NFS version or transport protocol is not supported

### ISSUE THREAD

**Issue by jayeshcmsminds** [geerlingguy/drupal-vm#1845]
I have tried to set the VBoxManage. I have installed vagrant 2.1.2 and VirtualBox too. But When run the command **"blt vm"** , after some successful packages installed it gives error ==> **"mount.nfs: requested NFS version or transport protocol is not supported"**

### Issue Type

 - Bug Report / Support Request

### Environment

```
vboxmanage --version ==> 5.2.18r124319
vagrant --version ===> Vagrant 2.1.2
```

###  OS

 - **Linux** (18.04)

### Summary
Following error I have faced:-

The following SSH command responded with a non-zero exit status.
Vagrant assumes that this means the command failed!

mount -o vers=3 192.168.97.1:/var/www/html/project-folder /var/www/project-folder

Stdout from the command:

Stderr from the command:

mount.nfs: requested NFS version or transport protocol is not supported

[Exec]  Exit code 1  Time 58.986s
[error]  Drupal VM failed to boot. Read Drupal VM's previous output for more information.

**Comment by geerlingguy** [comment]
It could be that your OS does not have NFS installed by default. Please see the linux setup instructions: http://docs.drupalvm.com/en/latest/getting-started/installation-linux/#ubuntu

**Comment by jesrig** [comment]
Thanks! No idea why I didn't have this in the past, but installing NFS locally fixed things.

**Comment by geerlingguy** [comment]
Thanks for posting the update! Hopefully this fix works for others using Linux!

**Comment by ldpm** [comment]
On the off chance someone else is finding this page even though their drupal vms had been happily chugging along for months until they weren't, check that that your host's NFS server is actually RUNNING, which turned out to be my problem:

`sudo systemctl status nfs-kernel-server`

I had tried to "clean up" some old vms (months ago) and removed the files but references to them were still in /etc/exports. This was not a problem until I went away for an extended break and therefore suspended all my vms and (importantly) shut down the host. When I came back, the NFS server exited with an error when the host restarted because it couldn't find the now missing directories.
